TradeItIosTicketSDK2/TradeItLinkedBrokerManager.swift

```diff
@@ -331,10 +319,38 @@ import PromiseKit
         linkedBrokersToRemove.forEach { linkedBrokerToRemove in
             self.removeBroker(linkedBroker: linkedBrokerToRemove)
         }
-        
+
+        // Sync accounts
+        self.linkedBrokers.forEach { localLinkedBroker in
+            remoteLinkedBrokers.first { remoteLinkedBroker in
+                localLinkedBroker.linkedLogin.userId == remoteLinkedBroker.userId
+            }.flatMap { remoteLinkedBroker in
+                syncAccounts(localLinkedBroker: localLinkedBroker, remoteLinkedBroker: remoteLinkedBroker)
```

Comment:
@guillaumedebavelaere I'm going to leave this as is for now. Yahoo doesn't need to check the `userToken`